src/pages/en/guides/integrations/tailwind.md

```diff
@@ -0,0 +1,187 @@
+---
+# NOTE: This file is auto-generated from 'scripts/generate-integration-pages.ts'
+#       and pulls content directly from the package’s README.
+#       DO NOT MAKE EDITS TO THIS FILE DIRECTLY, THEY WILL BE OVERWRITTEN!
+#       For corrections, please edit the package README at
+#       https://github.com/withastro/astro/tree/main/packages/integrations/tailwind
+
+layout: ~/layouts/MainLayout.astro
+title: '@astrojs/tailwind 💨'
+category: other
+i18nReady: false
+---
+
+This **[Astro integration][astro-integration]** brings [Tailwind's](https://tailwindcss.com/) utility CSS classes to every `.astro` file and [framework component](/en/core-concepts/framework-components/) in your project, along with support for the Tailwind configuration file.
```

Comment:
```suggestion
This **[Astro integration][astro-integration]** brings [Tailwind's](https://tailwindcss.com/) utility CSS classes to every `.astro` file and [framework component](/en/core-concepts/framework-components/) in your project, along with support for the Tailwind configuration file.
```
But, now I'm noticing this framework-component is a relative link... which probably doesn't work in the README itself? (Not something to correct here, necessarily... but just another reminder that we're going to have to make sure we know what we're doing with these links, defined outside of docs and leading to docs!)

Reply:
Aha! I DID fix them then. Just the one above is a format my fix didn't handle. They should be absolute in the READMEs but I'm making them relative when generating the docs versions.